ui/apps/platform/cypress/integration/dashboard.test.js

```diff
@@ -14,60 +14,96 @@ import withAuth from '../helpers/basicAuth';
 import { visitMainDashboard } from '../helpers/main';
 import baseSelectors from '../selectors/index';
 
+// For future redesign of main dashboard, separate tests for these requests into a separate file.
+
+const alertsSummaryCountsByCluster0 = { groups: [] };
+
+const alertsSummaryCountsByCluster1 = {
+    groups: [
+        {
+            group: 'Kubernetes Cluster 0',
+            counts: [
+                { severity: 'LOW_SEVERITY', count: '2' },
+                { severity: 'MEDIUM_SEVERITY', count: '1' },
+                { severity: 'CRITICAL_SEVERITY', count: '0' },
+            ],
+        },
+    ],
+};
+
+const alertsSummaryCountsByCluster2 = {
+    groups: [
+        {
+            group: 'Kubernetes Cluster 0',
+            counts: [
+                { severity: 'LOW_SEVERITY', count: '2' },
+                { severity: 'MEDIUM_SEVERITY', count: '1' },
+            ],
+        },
+        {
+            group: 'Kubernetes Cluster 1',
+            counts: [
+                { severity: 'HIGH_SEVERITY', count: '10' },
+                { severity: 'CRITICAL_SEVERITY', count: '5' },
+            ],
+        },
+    ],
+};
+
 describe('Dashboard page', () => {
     withAuth();
 
-    it('should select item in nav bar', () => {
-        cy.visit(dashboardUrl);
+    // Skip tests for elements which seem less likely to have a counterpart in future design.
+
+    it('should select item in left nav', () => {
+        visitMainDashboard();
+
         cy.get(selectors.navLink).should('have.class', 'pf-m-current');
     });
 
     it('should display system violations tiles', () => {
         cy.intercept('GET', api.alerts.countsByCluster, {
-            fixture: 'alerts/countsByCluster-single.json',
-        });
-
+            body: alertsSummaryCountsByCluster1,
+        }).as('getAlertsCountsByCluster');
         visitMainDashboard();
+        cy.wait('@getAlertsCountsByCluster');
 
-        cy.get(selectors.sectionHeaders.systemViolations).next('div').children().as('riskTiles');
-
-        cy.get('@riskTiles').spread((aCritical, aHigh, aMedium, aLow) => {
-            cy.wrap(aLow).should('have.text', '2Low');
-            cy.wrap(aMedium).should('have.text', '1Medium');
-            cy.wrap(aHigh).should('have.text', '0High');
-            cy.wrap(aCritical).should('have.text', '0Critical');
-        });
+        cy.get(`${selectors.severityTile}:contains("2Low")`);
+        cy.get(`${selectors.severityTile}:contains("1Medium")`);
+        cy.get(`${selectors.severityTile}:contains("0High")`);
+        cy.get(`${selectors.severityTile}:contains("0Critical")`);
     });
 
     it('should not navigate to the violations page when clicking the critical severity risk tile', () => {
+        // For future design of main dashboard: Why not? A link is valid, even if no violations right now.
```

Comment:
For sure, it seems like there is value in users having a link to "High Severity Violations", even if the current total is zero. It would make the UI and the code more consistent to just always render a link as well.